src/NUnitFramework/framework/Internal/Filters/ValueMatchFilter.cs

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
 
-using System;
-using System.Collections.Generic;
+#nullable enable
```

Comment:
I'm having a hard time submitting this comment as a suggestion, but I think this adds duplicate `#nullable enable` in the file (on line 7), though for consistency it might be good to remove that one and keep this to have them all above the `using`s